ACPs/103-dynamic-fees/README.md

```diff
@@ -122,7 +124,7 @@ A full reference implementation has not been provided yet. It must be provided p
 
 ## Security Considerations
 
-The existing fixed fee mechanism on the X-Chain and P-Chain have led to periods of instability when larger transactions are accepted. Metering these transactions according to their computational cost or "gas" usage will remove those periods of instability at the cost of higher fees for users.
+The current fixed fee mechanism on the X-Chain and P-Chain does not robustly handle spikes in load. Switching these chains to a dynamic fee mechanism will stabilize the load over the long-term.
```

Comment:
> will stabilize the load over the long-term

will ensure that load is properly priced given allotted processing capacity.

****

I don't think this is capable of "stabilizing load" (nor do I think that is the goal).